styles/style.css

```diff
@@ -0,0 +1,1088 @@
+*{
```

Comment:
Ondanks dat je in dit project slechts CSS kunt gebruiken is dit bestand wel erg lang. Je had er wellicht voor kunnen kiezen om de verschillende "features" op te splitsen over meerdere CSS bestanden.